packages/oid4vci-demo-frontend/package.json

```diff
@@ -12,7 +12,7 @@
     "@sphereon/ssi-sdk.qr-code-generator": "0.17.6-unstable.6",
     "@sphereon/ssi-sdk.siopv2-oid4vp-rp-rest-client": "0.17.6-unstable.6",
     "@sphereon/ssi-types": "0.17.6-unstable.6",
-    "@sphereon/ui-components.ssi-react": "0.1.3-next.13",
+    "@sphereon/ui-components.ssi-react": "file://C:\\Git\\Sphereon-Opensource\\UI-Components\\packages\\ssi-react\\sphereon-ui-components.ssi-react-109.1.2.tgz",
```

Comment:
TODO fix after marge & publish the UI components branch

Reply:
updated with the new release